### ISSUE TITLE
build fails on armhf, armv7 and x86 (32bit)

### ISSUE THREAD

**Issue by tuananh** [google/osv-scanner#101]
```
../go/github.com/spdx/gordf@v0.0.0-20201111095634-7098f93598fb/rdfloader/xmlreader/xmlreader.go:213:27: WHITESPACE (untyped int constant 4294977024) overflows int
>>> ERROR: osv-scanner: build failed
```

i'm trying to add osv-scanner to aports for alpinelinux.

the pipelines are mostly fine except for armhf, armv7 and x86 with the above error

any idea why?

https://gitlab.alpinelinux.org/tuananh/aports/-/pipelines/147834

**Comment by tuananh** [comment]
the issue is from this dependency https://github.com/spdx/gordf/issues/7

**Comment by oliverchang** [comment]
This is fixed by #104, and should be released next week.

**Comment by another-rex** [comment]
Fixed in v1.1.0 release.
